terraform/modules/credhub/variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "stack_description" {}
+
+variable "credhub_cidr" {
```

Comment:
This appears to be unused.

Reply:
yeah, we were originally going to add a ELB to this, but we don't need it anymore good catch.